PARAM_DIFFERENCES.md

```diff
@@ -0,0 +1,57 @@
+## Parameter Differences
+
+### RDSE
+| HTMCore    |                                                                                                                                                                                                                                                                                            |              | Nupic CPP  |                                                                                                                                                                                                                                                                                                                 |         |
```

Comment:
not Nupic CPP (that was our former name), ratner NuPIC, or Numenta's NuPIC

Reply:
Done 👍